jobserv_worker.py

```diff
@@ -26,6 +27,7 @@
 from configparser import ConfigParser
 from multiprocessing import cpu_count
 
+from psutil import virtual_memory
```

Comment:
While you're right, `requests`, is common, `psutil` is pretty common too.  That said, I agree with you that I'd like to have this be more like an executable.  The moment I couldn't run this script due to the use of `requests` it was a bit jarring. It would be nice if we could convert this to a go executable.  So there are all deps are vendors. Or, to save time we could make the download a [pex](https://github.com/pantsbuild/pex) file. In other words installation of any other dependency is jarring.

Reply:
i'd really love to move this to golang some day. running this worker on rpi3's take's a surprising amount of compute resources. And ... we'd get more type-safety, and a static binary, etc etc. 

This one is really high on my list of "if i had it to do all over again" :)

Reply:
I removed the changes and added a helpful note to anyone attempting to run this on a non-linux system.  I figure one could assume this will work on their box windows/Mac/linux etc since it's written in python, but there are real issues at least with the Mac I've found.  Docker  in Docker doesn't work rootless for one, and I was unaware initially that this script will run additional shell scripts.  I think it would be good to also add a docker check because the worker will hit the error only after having a build triggered. It would be a better experience if the user was notified earlier on. I also think that using `#!/usr/bin/env python3` is beneficial in this script as it doesn't assume too much about the users environment. These changes can be approached in a different PR though if desired.